s3fs/core.py

```diff
@@ -1042,7 +1042,7 @@ async def _info(self, path, bucket=None, key=None, refresh=False, version_id=Non
                     "size": out["ContentLength"],
                     "name": "/".join([bucket, key]),
                     "type": "file",
-                    "StorageClass": "STANDARD",
+                    "StorageClass": out.get("StorageClass", "STANDARD"),
```

Comment:
Is this ever missing?

Reply:
I saw StorageClass could be different from standard when set on write

Reply:
OK